icon-validator/validate-icon.c

```diff
@@ -49,18 +49,26 @@ validate_icon (const char *arg_width,
       return 1;
     }
 
-  max_width = g_ascii_strtoll (arg_width, NULL, 10);
-  if (max_width < 16 || max_width > 4096)
+  if (g_str_equal (name, "svg"))
```

Comment:
Isn't this the wrong way around? I.e. *only* checks size limits for svgs?

Reply:
Yes. I probably thought of using `g_strcmp0()` but changed my mind...